rspirv/lift/storage.rs

```diff
@@ -0,0 +1,59 @@
+use crate::{
+    sr::{
+        storage::{Storage, Token},
+    },
+};
+
+use std::{
+    borrow::Borrow,
+    collections::hash_map::{Entry, VacantEntry, HashMap},
+    hash::BuildHasherDefault,
+};
+
+use fxhash::FxHasher;
+
+
+pub struct LiftStorage<T, L = Token<T>> {
```

Comment:
Can we please have documentation for public APIs?

Reply:
yes, sure 😅